packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js

```diff
@@ -49,20 +51,25 @@ function ActionDropdown(props) {
 			{hideLabel ? null : <span>{label}</span>}
 		</span>
 	);
-
 	const style = link ? 'link' : bsStyle;
+	const onItemSelect = (object, event) => {
```

Comment:
if you don't need 'this' don't use arrow function, please use function instead.
https://github.com/Talend/ui/wiki/JavaScript-coding-style#lambda-aka-arrow-function